src/Configuration/ValueObject/ExportDatabaseConfig.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Configuration\ValueObject;
+
+use Keboola\DbExtractorConfig\Exception\PropertyNotSetException;
+
+class ExportDatabaseConfig
```

Comment:
Mam poznamku k pomenovaniu, ... ty si tiez o tom pisal:

Podla mna je `ExportConfig` dobry nazov, lebo ten objekt riesi co/kde/ako sa ma _exportovať_

Tuto triedu by som odporucal nazvat `DbConfig` (zodpoveda tomu `DbNode`), `DatabaseConfig` alebo `ConnectionConfig` ... lebo obsahuje informácie o tom k akej _databáze_ sa _pripojit_.

Nevidim dovod ani potrebu prefixovat to s `Export`, trieda je v namespace `DbExtractorConfig`.

Reply:
no podle toho co píšeš by teda měl ExportConfig být spíše pojmenovaný jako `TableConfig`
takhle jsem to pojmenoval aby byla třída pojemnovaná "stejným" stylem

Reply:
`TableConfig` by bolo mozno lepsie ... no stale mi to slovo `Export` v tom nazve nedava zmysel, ... takze, nechcem to na tvojom zvazeni.

Reply:
no když se nad tím zamyslím tak opravdu to bude lepší přejmenovat, až jednou budeme dělat config který bude obsahovat všechny tyhle třídy tak tam ten Export bude opravdu opruz a přejmenujeme jen ten ExportConfig na TableConfig